draft-ietf-openpgp-pqc.md

```diff
@@ -1283,341 +1051,234 @@ TBD (SLH-DSA-SHAKE-256s) |  SHA3-512      | 14
-when using a sponge-based construction like KMAC. The construction using KMAC
-ensures a standardized domain separation. In this case, the processed message
-is then the concatenation of any number of key shares.
-
-More precisely, for a given capacity `c` the indifferentiability proof shows
-that assuming there are no weaknesses found in the Keccak permutation, an
-attacker has to make an expected number of `2^(c/2)` calls to the permutation
-to tell KMAC from a random oracle. For a random oracle, a difference in only a
-single bit gives an unrelated, uniformly random output. Hence, to be able to
-distinguish a key `K`, derived from shared keys `K1` and `K2` (and ciphertexts
-`C1` and `C2`) as
-
-    K = KMAC(domainSeparation, counter || K1 || C1 || K2 || C2 || fixedInfo,
-             outputBits, customization)
-
-from a random bit string, an adversary has to know (or correctly guess) both
-key shares `K1` and `K2`, entirely.
-
-The proposed construction in {{kem-key-combiner}} preserves IND-CCA2 of any of
-its ingredient KEMs, i.e. the newly formed combined KEM is IND-CCA2 secure as
-long as at least one of the ingredient KEMs is. Indeed, the above stated
-indifferentiability from a random oracle qualifies Keccak as a split-key
-pseudorandom function as defined in {{GHP18}}. That is, Keccak behaves like a
-random function if at least one input shared secret is picked uniformly at
-random. Our construction can thus be seen as an instantiation of the IND-CCA2
-preserving Example 3 in Figure 1 of {{GHP18}}, up to some reordering of input
-shared secrets and ciphertexts. In the random oracle setting, the reordering
-does not influence the arguments in {{GHP18}}.
+For the key combination in {{kem-key-combiner}} this specification limits itself to the use of KMAC.
+The sponge construction used by KMAC was proven to be indifferentiable from a random oracle {{BDPA08}}.
+This means, that in contrast to SHA2, which uses a Merkle-Damgard construction, no HMAC-based construction is required for key combination.
+Except for a domain separation it is sufficient to simply process the concatenation of any number of key shares when using a sponge-based construction like KMAC.
+The construction using KMAC ensures a standardized domain separation.
+In this case, the processed message is then the concatenation of any number of key shares.
+
+More precisely, for a given capacity `c` the indifferentiability proof shows that assuming there are no weaknesses found in the Keccak permutation, an attacker has to make an expected number of `2^(c/2)` calls to the permutation to tell KMAC from a random oracle.
+For a random oracle, a difference in only a single bit gives an unrelated, uniformly random output.
+Hence, to be able to distinguish a key `K`, derived from shared keys `K1` and `K2` (and ciphertexts `C1` and `C2`) as
+
+    K = KMAC(domainSeparation, counter || K1 || C1 || K2 || C2 || fixedInfo, outputBits, customization)
```

Comment:
This needs to persist the line break as it's in a preformatted block

Reply:
Oh yes, I undid that change again.